packages/catalog-process/test/catalogService.integration.test.ts

```diff
@@ -151,6 +178,12 @@ describe("database test", async () => {
     await postgresDB.none("TRUNCATE TABLE agreement.events RESTART IDENTITY");
   });
 
+  afterAll(async () => {
```

Comment:
Not that I noticed - still, I found some issues with `testcontainers` sometimes and maybe they could be related to this?
For example:
- tests sometimes fail with timeout on the CI, even though they work locally
- I tried to move to `commons-test` the functions that start containers, but for some reason tests cannot find the containers references at runtime (they are `undefined`) even though everything compiles correctly

Maybe these are issues related to containers stopping?

Reply:
@galales no downsides in adding the stop instructions there. It should not be needed, but we can keep those for extra safety.

Here is my point in details:
- each test suite creates/uses/stops/removes its own containers. There is no reuse between different test suites of different packages, according to how we have structured the test suites.
- the stop and deletion are handled automatically by the library, so there should be no need to add those.
- No matter if the tests pass or fail (based on the `expect` conditions), the container are stopped and removed correctly even without doing anything in `afterAll`. During local development it might happen that in some rare cases the containers remain stopped but are not removed. However I am not able to reproduce the issue to find the specific cause. Anyways, in my opinion this happens when the test suite is re-run many times or exited with some external problems, but in those cases the `afterAll` method is not executed at all, so stopping the containers there don't help.
- As far as I understand, the only difference we have by adding the stop instructions in `afterAll` is that we wait for the containers to be stopped/removed before exiting the test suite, otherwise that operation will not be waited. ([Reference](https://node.testcontainers.org/features/containers/#stopping-a-container))